src/components/fieldset/fieldset.yaml

```diff
@@ -1,3 +1,42 @@
+params:
+- name: describedBy
+  type: string
+  required: false
+  description: Text or element id to add to the `aria-describedby` attribute to provide description of the group of fields for screenreader users.
+- name: legend
+  type: object
+  required: false
+  description: Options for the legend
+  params:
+  - name: text
+    type: string
+    required: true
+    description: If `html` is set, this is not required. Text to use within the legend. If `html` is provided, the `text` argument will be ignored.
+  - name: html
+    type: string
+    required: true
+    description: If `text` is set, this is not required. HTML to use within the legend. If `html` is provided, the `text` argument will be ignored.
+  - name: classes
+    type: string
+    required: false
+    description: Classes to add to the legend container.
+  - name: isPageHeading
+    type: boolean
+    required: false
+    description: Whether the legend also acts as the heading for the page.
+- name: classes
+  type: string
+  required: false
+  description: Classes to add to the fieldset container.
+- name: attributes
+  type: object
+  required: false
+  description: HTML attributes (for example data attributes) to add to the fieldset container.
+- name: caller
+  type: nunjucks-block
+  required: false
+  description: Not strictly a parameter but <a href="https://mozilla.github.io/nunjucks/templating.html#call">Nunjucks code convention</a>. Using a `call` block enables you to call a macro with all the text inside the tag. This is helpful if you want to pass a lot of content into a macro. To use it, you will need to wrap the entire fielset component in a `call` block. See <a href="https://github.com/alphagov/govuk-frontend/blob/master/src/components/checkboxes/template.njk#L86">checkboxes component</a> for an example.
```

Comment:
We're using html here for the links. We're also using ` backticks for option names, which is a markdown convention. We don't have to do anything with the backticks, but it's probably best to run them through a renderer to get <code> blocks. In which case maybe we should just use markdown for the links too?

Reply:
Good shout. I was avoiding markdown since I only needed to use HTML in this one place for the component options and we discovered we'll need to do some extra processing as part of the build in the Design System to render markdown from here. But since we've got the code blocks too then making it all markdown makes sense 🙂